contents.json

```diff
@@ -652,7 +652,14 @@
       "category": "third-party-guides",
       "description": "Memory game implementation that also teaches Swift.",
       "homepage": "https://github.com/Sajjon/SwiftIntro"
-    }, {
+    }, 
+    {
+     "title": "Ray Wenderlich Tutorials, Videos, Podcasts and books",
+     "category": "third-party-guides",
+     "description": "Hands down Swift Dev top resource",
+     "homepage": "http://raywenderlich.com"
```

Comment:
@bbookman Could you replace http://raywenderlich.com with https://www.raywenderlich.com?